internal/config/default_config.go

```diff
@@ -2,15 +2,13 @@ package config
 
 import (
 	"time"
-
-	"github.com/spf13/viper"
 )
 
 var (
 	Server = map[string]string{
 		"name":      "meshery-meshsync",
 		"port":      "11000",
-		"version":   viper.GetString("BUILD"),
+		"version":   "latest",
```

Comment:
We do want the actual build tag, though…

Reply:
This is just to set something, this will be overridden by https://github.com/meshery/meshsync/pull/133/commits/257bf3b5ac2bbf44fadc677b79a3e6288c787528#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R48